app/models/issue.js

```diff
@@ -1,34 +0,0 @@
-import DS from 'ember-data';
-import { computed } from '@ember/object';
-
-export default DS.Model.extend({
```

Comment:
Can we keep the model? Seems like this is a feature that we *have to use* in Ember.js.

Reply:
No need ATM, probably will re-add it when I add support for GitLab.
However, learning from the past, I will try to keep dependency as simple as possible. Adding ember data was adding uneeded complexity, for me.

Reply:
The issue model was originally used for GitHub using v3 Api.

Reply:
Please keep the models. We will need this level of sophistication.
Supporting multiple hosters means the app must declare its own data structure which supports a subset of each hosters features.